src/components/_cards/BridgeCard/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Heading, Stack, Center, Text } from "@chakra-ui/react"
+import { BaseButton } from "components/_buttons/BaseButton"
+import { ExternalLinkIcon, TimerIcon } from "components/_icons"
+import React from "react"
+import { TransparentCard } from "../TransparentCard"
+import { Link } from "components/Link"
+import { InputAmount } from "./InputAmount"
+import { EthereumAddress } from "./EthereumAddress"
+import { InputCosmosAddress } from "./InputCosmosAddress"
+
+export const BridgeCard: React.FC = () => (
+  <TransparentCard
+    maxW="432px"
+    w="full"
+    boxShadow="0 0 0 2px #6C4ED9"
```

Comment:
Let's not hardcode color values if we can help it. If it doesn't exist in `./src/theme/colors.ts`, we should add it there. Could create a `shadows` object 👀